test/fakeintake/server/body.go

```diff
@@ -45,12 +45,12 @@ func getConnectionsResponse() []byte {
 }
 
 // getResponseFromURLPath returns the appropriate response body to HTTP request sent to 'urlPath'
-func getResponseFromURLPath(urlPath string) httpResponse {
-	var defaultResponse = httpResponse{
+func (fi *Server) getResponseFromURLPath(urlPath string) httpResponse {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
You could rename this to `defaultResponseFromURLPath` and keep it static, adding a new `Server` struct function to `server.go` called `getResponseFromURLPath` that first read-locks the response mutex and check for an override, if not uses the default response.

Reply:
As discussed, we are going to add these custom static responses to the same `server.responseOverrides` map, so that we can reduce the three-tiered response behavior to just two.

In the server constructor, a `responseOverrides` map is created with these existing custom responses, with the ability for the user to create additional response overrides. I've also factored out the `responseOverrides` initialization so that we can add a "reset" function in the future to re-initialize the map.